libs/client/ui/pages/src/lib/profile/friends/create-conversation-modal/create-conversation-modal.tsx

```diff
@@ -0,0 +1,223 @@
+        setIsLoading(true);
+
+        const query = e.detail.value?.trim().toLowerCase();
+        if (!query) {
+          fetchFriends();
+          return setIsLoading(false);
+        }
+
+        friendsStore
+          ?.searchFriend({ username: query, onlyConversationsless: true })
+          .then((result) => {
+            setSearchResult(
+              result.map((r) => ({
+                ...r,
+                hasConversation: false,
+              }))
+            );
+            setIsLoading(false);
+          })
+          .catch(() => {
+            setSearchResult([]);
+            setIsLoading(false);
+          });
+      };
+
+      const fetchFriends = () => {
+        friendsStore
+          ?.fetchFriends({ onlyConversationsless: true })
+          .then((result) =>
+            setSearchResult(
+              result.map((u) => ({
+                id: u.id,
+                username: u.username,
+                profilePicture: u.profilePicture,
+                hasConversation: false,
+              }))
+            )
+          )
+          .catch(() => setSearchResult([]));
+      };
+
+      const handleCreateConversation = (participantId: User['id']) => {
+        chatStore
+          ?.createConversation({ participantId })
+          .then((conversation) => {
+            setSearchResult((prev) =>
+              prev.map((r) =>
+                r.id === participantId ? { ...r, hasConversation: true } : r
+              )
+            );
+          })
+          .catch(() => {
+            presentDefaultErrorToast();
+          });
+      };
+
+      useEffect(() => {
+        fetchFriends();
+      }, []);
+
+      return (
+        <IonModal
+          trigger={trigger}
+          ref={modalRef}
+          canDismiss
+          className="create-conversation-modal"
+        >
+          <IonHeader mode="ios" collapse="condense" className="ion-no-border">
+            <IonToolbar>
+              <IonTitle>Neuer Chat</IonTitle>
+              <IonButtons slot="start">
+                <IonButton onClick={() => dismissAddModal()}>Zurück</IonButton>
+              </IonButtons>
+            </IonToolbar>
+            <IonSearchbar
+              mode="ios"
+              value={searchText}
+              onIonChange={(e) => handleSearch(e)}
+              debounce={250}
+              placeholder="Freunde für neuen Chat suchen"
+            />
+          </IonHeader>
+          <IonContent className="create-conversation-modal__content">
+            <IonList className="friend-user-list">
+              {isLoading || searchResult.length > 0
+                ? searchResult.map((user) => {
+                    return (
+                      <IonRow key={user.id}>
+                        <IonCol size="auto" className="username-avatar-col">
+                          <IonItem
+                            key={user.id}
+                            routerLink={`/users/${user.id}`}
```

Comment:
wo söt des ane verlinken

Reply:
Zum competitor detail

Reply:
competitor detail hot da username in da url

Reply:
shit stimmt

Reply:
aba des mitm username in da url kummt ersch mit #277